src/frontendproxy/Dockerfile

```diff
@@ -1,8 +1,7 @@
 # Copyright The OpenTelemetry Authors
 # SPDX-License-Identifier: Apache-2.0
 
-
-FROM envoyproxy/envoy:v1.28-latest
+FROM envoyproxy/envoy:dev-698abe98131fa5c64904082407878d1fcc62cb88
```

Comment:
Would you mind waiting for the stable release, unless using the `dev` one?

Reply:
@rlankfo I saw that `v1.29.0` was released, would you mind updating the PR and adding an entry in the changelog for this?

Reply:
absolutely! thanks for the heads up.